src/modules/order/payment.service.ts

```
@@ -19,29 +19,114 @@ export class PaymentService {
+
+            transactionId = session.id;
+            paymentIntentId = typeof session.payment_intent === 'string' ? session.payment_intent : session.payment_intent?.id || null;
+
+            // const pi = 
+
+            switch (event.type) {
+                // case 'checkout.session.completed':
+                case 'payment_intent.succeeded':
+                    paymentStatus = 'success';
+
+                    const chrge = (event as any).data.object?.charges?.data[0]
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)